station-flattening-v0/src/main/java/net/modificationstation/stationapi/mixin/flattening/client/MixinWorldRenderer.java

```diff
@@ -35,6 +43,26 @@ private int changeMaxBlockHeight(int value) {
 	private int changeSectionCount(int value) {
 		return level == null ? value : level.countVerticalSections();
 	}
+	
+	@Inject(method = "playLevelEvent", at = @At("HEAD"), cancellable = true)
+	private void changeBlockLimit(PlayerBase i, int type, int x, int y, int z, int data, CallbackInfo info) {
+		if (type != 2001) return;
+		info.cancel();
```

Comment:
Couldn't this also be done with a couple of `@ModifyConstant` instead? If `meta, id` order is preserved by the caller classes.